packager/react-packager/src/Server/index.js

```diff
@@ -739,7 +741,10 @@ class Server {
           if (!original) {
             return frame;
           }
+          const relativePath = original.source.replace(appRoot, '');
           return Object.assign({}, frame, {
+            fileName: path.basename(relativePath),
+            filePath: path.dirname(relativePath),
```

Comment:
other properties in this object (`file`, `lineNumber`, `column`) are pretty common in the other JS engines' notions of "stack frame". These 2, on the other hand, seem to be derived from the `file`. I think if we want to have shorter file names, instead of modifying the concept of the stack frame we should send `projectRoot` as one of the pieces of data along with stack - see line 755 below